tests/bam/bam_pb.robot

```diff
@@ -795,7 +795,7 @@ BEPB_DIMENSION_TRUNCATE_TABLE
 	Kindly Stop Broker  True
 
 BA_RATIO_NUMBER_BA_4_SERVICE
-	[Documentation]	With bbdo version 3.0.1, a BA of type 'ratio number' with 4 serv
+	[Documentation]	With bbdo version 3.0.1, a BA of type 'ratio number' with 4 services, the BA should change it's status based on the number definded for the warning and critical state of its service.
```

Comment:
Be careful, a ratio number ba counts kpi in critical. The more we have critical kpis, the more critical is the ba.
We don't mind of warning services.